main/createUiDefinition.json

```diff
@@ -110,6 +110,13 @@
             },
             "visible": true
           },
+          {
+            "name": "enableIPForwarding",
+            "type": "Microsoft.Common.CheckBox",
+            "label": "IP forwarding",
+            "defaultValue": true,
```

Comment:
Should be false

Reply:
If we would like to avoid it in UI, we can just hardcoded as I did in first commit.